README.md

```diff
@@ -31,6 +31,22 @@ Once you have your `APP ID` and your tokens, you can fill out the forum at https
 
 It may take a few days for you to get approved.
 
+### Slack
+
+In order to collect data from Slack's API, you need to setup a Slack application. To start you can go to https://api.slack.com/apps/ and click the `Create New App` button. Give your app a name and assign it to your workspace.
+
+On the next page, click the `Permissions` section underneath `Add features and functionality`. Scroll down to the `Scopes` section and click `Add an OAuth Scope`. This should open up a downdown menu.
```

Comment:
my preference is not to use backticks on button names or menu items - bold or plain

Reply:
```suggestion
On the next page, click the `Permissions` section underneath `Add features and functionality`. Scroll down to the `Scopes` section and click `Add an OAuth Scope`. This should open up a dropdown menu.
```

Reply:
Cool sounds good, I'll update all of them